examples/Example0_LibrarylessCommunication/Example0_LibrarylessCommunication.ino

```diff
@@ -90,7 +90,7 @@ void loop()
     static unsigned eventCounter = 0;
     if (++eventCounter > 25)
     {
-        return;
+        while(1);
```

Comment:
Just curious why?

Reply:
The debugger will land in one place, instead of jumping inside and outside of `loop()`.

It's interesting, because `return` would be preferable for something like Particle, because they need to exit loop to manage there Wi-Fi connections, but most microcontrollers just spin on `loop`, something akin to:

```c
while (1) {
    loop();
}
```

I would probably be best to print a message, have a 10 second `delay`, and then `return`.

I'll update accordingly...